transcom/mymove
# B 19533 add spouse pro gear weights page integration

## [19533](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A940551)

## Summary

In this PR you should now see the actual pro gear and Spouse pro gear weights, departure date, and spouse pro gear entitlements weight on the Review weights page.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create a Full or Partial Move with a PPM
2. Login as a SC, edit and submit move
3. Login as customer and click "Upload Documents"
4. Enter in all information and take note of actual pro gear weights and departure date
5. Sign and Save move
6. Login as a Closeout SC and click "Review Weights Page"
7. You should now see the PPM actual pro gear weights, departure date and pro gear weight entitlements

### Frontend



## Screenshots
<img width="1042" alt="Screenshot 2024-05-29 at 1 56 25 PM" src="https://github.com/transcom/mymove/assets/146969726/7db78d04-8dc2-48a0-9a90-8b9d4faceab9">
